gateway-service/src/main/java/org/zowe/apiml/gateway/controllers/AuthController.java

```diff
@@ -80,6 +85,7 @@ public class AuthController {
     public static final String ALL_PUBLIC_KEYS_PATH = PUBLIC_KEYS_PATH + "/all";
     public static final String CURRENT_PUBLIC_KEYS_PATH = PUBLIC_KEYS_PATH + "/current";
     public static final String OIDC_TOKEN_VALIDATE = "/oidc-token/validate"; // NOSONAR
+    public static final String OIDC_WEBFINGER_PATH = "/webfinger";
```

Comment:
Maybe it would good to add some prefix to identify it's OIDC related, so like `/oidc/webfinger`, what do you think?